readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -479,30 +466,36 @@ class PdfBuilder(BaseSphinx):
 
     """Builder to generate PDF documentation."""
 
-    type = 'sphinx_pdf'
-    sphinx_build_dir = '_build/latex'
+    sphinx_build_dir = "_readthedocs/pdf"
+    sphinx_builder = "latex"
     pdf_file_name = None
 
     def build(self):
-        self.clean()
-        cwd = os.path.dirname(self.config_file)
-
-        # Default to this so we can return it always.
         self.run(
             *self.get_sphinx_cmd(),
-            '-b',
-            'latex',
+            # New arguments for standardization
```

Comment:
This comment could use work. It doesn't tell me what it actually does or why. Standardization with what?  Seems like it's enabling full tracebacks? https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-T

Reply:
He he, this was more an internal to comment in the PR description, that I finally forget removing and commenting in the description 😓 . I added this arguments to match the ones we are using in all the other Sphinx commands. I'm happy to rever this, but I think it's correct to standardize them on all the commands. We have added some arguments to the other commands over time but we haven't add those in this one.

Reply:
Let me know if it's fine keeping them or I should remove them for now.